pkg/cmd/cluster/bind/bind.go

```diff
@@ -102,21 +107,27 @@ func runBind(opts *Options) error {
 		opts.selectedKafka = cfg.Services.Kafka.ClusterID
 	}
 
-	api := connection.API()
-	kafkaInstance, _, error := api.Kafka().GetKafkaById(context.Background(), opts.selectedKafka).Execute()
+	api := apiConnection.API()
+	kafkaInstance, _, err2 := api.Kafka().GetKafkaById(context.Background(), opts.selectedKafka).Execute()
```

Comment:
```suggestion
	kafkaInstance, _, err := api.Kafka().GetKafkaById(context.Background(), opts.selectedKafka).Execute()
```

nit: Why create new error?

Reply:
Lint issues

Reply:
Lint issues - creating new instance here